scenarios/tracking/README.md

```diff
@@ -2,4 +2,131 @@
+
+## Technology
+Multi-object-tracking (MOT) is one of the hot research topics in Computer Vision, due to its wide applications in autonomous driving, traffic surveillance, etc. It builds on object detection technology, in order to detect and track all objects in a dynamic scene over time, without any prior knowledge about the appearance and number of target objects. Inferring target trajectories correctly across successive image frames remains challenging: occlusion happens when objects overlap; the number of and appearance of objects can change. Compared to object detection algorithms, which aim to output rectangular bounding boxes around the objects, MOT algorithms additionally associated an ID number to each box to identify that specific object across the image frames. 
+
+As seen in the figure below ([Ciaparrone, 2019](https://arxiv.org/pdf/1907.12740.pdf)), a typical multi-object-tracking algorithm performs part or all of the following steps:
+* Detection: Given the input raw image frames (step 1), the detector identifies object(s) on each image frame as bounding box(es) (step 2).
+* Feature extraction/motion prediction: For every detected object, visual appearance and motion features are extracted (step 3). Sometimes, a motion predictor (e.g. Kalman Filter) is also added to predict the next position of each tracked target. 
+* Affinity: The feature and motion predictions are used to calculate similarity/distance scores between pairs of detections and/or tracklets, or the probabilities of detections belonging to a given target or tracklet (step 4). 
+* Association: Based on these scores/probabilities, a specific numerical ID is assigned to each detected object as it is tracked across successive image frames (step 5).
+
+<p align="center">
+<img src="./media/figure_motmodules.jpg" width="600" align="center"/>
+</p> 
+
+
+**Online vs offline(batch tracking)**   
+
+In general, MOT algorithms can be divided into online tracking and offline tracking, which differ at the data association step. In online tracking, only the observations up to the current frame are taken into consideration when processing a new frame: typically, the new detections in the new frame are associated with tracks generated previously from the previous frame (see figure below- #TODO check source of img with @Casey), thus existing tracks are extended or new tracks are created. In offline tracking, all observations in a batch of frames are considered globally (see figure below), in that they are linked together into tracks by obtaining a global optimal solution. In this way, offline tracking can perform better with tracking issues such as long-term occlusion, or similar targets that are spatially close. However, offline tracking is slow not suitable for online tasks, such as for autonomous driving. Recently, research has focused on online tracking algorithms, which have reached the performance of offline-tracking, while still maintaining high inference speed. 
```

Comment:
Please remove all "TODO" statements etc in the two files so that it is ready to go into staging.